includes/admin/views/html-order-track-trace-table.php

```diff
@@ -61,8 +61,10 @@
           $order            = wc_get_order($order_id);
           $returnShipmentId = $order->get_meta(WCMYPA_Admin::META_RETURN_SHIPMENT_IDS);
 
+          $order_id = WPO\WC\MyParcel\Compatibility\Order::get_id($order);
+
           WCMYPA_Admin::renderAction(
-              admin_url("admin-ajax.php?action=$action&request=$getLabels&shipment_ids=$shipment_id&return_shipment_id=$returnShipmentId"),
+              admin_url("admin-ajax.php?action=$action&request=$getLabels&order_ids=$order"),
```

Comment:
Je stopt hier de hele order erin ipv het order id, heb je dit getest? Waarom vallen `shipment_ids` en `return_shipment_id` weg?

Reply:
De reden dat ik `shipment_ids` en `return_shipment_id` weg zijn uit de url is omdat nu de button exact dezelfde behavior vertoont als de andere print buttons.

De `$order` en `$order_id` is een foutje van mij.